make/modules/jdk.jartool/Java.gmk

```diff
@@ -25,3 +25,5 @@
 
 DISABLED_WARNINGS_java += missing-explicit-ctor
 JAVAC_FLAGS += -XDstringConcat=inline
+JAVAC_FLAGS += --enable-preview
```

Comment:
This change is no longer needed.  It's not a preview API.

Reply:
Unfortunately patterns in switch statements are still in preview. This flag and participation in preview can be removed once they are finalised.

Reply:
Thanks for the clarification.   I forgot about that.